core/src/test/java/org/infinispan/distribution/rehash/ConsistencyStressTest.java

```diff
@@ -83,14 +83,7 @@ protected void createCacheManagers() throws Throwable {
 
    public void testConsistency() throws Throwable, InterruptedException {
       // create an executor...
-      ExecutorService executorService = Executors.newFixedThreadPool(NUM_NODES * WORKERS_PER_NODE, new ThreadFactory() {
-         int i = 0;
-
-         @Override
-         public synchronized Thread newThread(Runnable r) {
-            return new Thread(r, "Worker-" + i++);
-         }
-      });
+      ExecutorService executorService = Executors.newFixedThreadPool(NUM_NODES * WORKERS_PER_NODE, getTestThreadFactory("Worker"));
```

Comment:
you could use fork() for this case IMO. Also, you have the ThreadFactory unused import